test/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:12
+
+ENV password=notpassword
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- ea06001c-78c6-4b1b-9ab6-f530023ccc82 -->